server/service/client_policies.go

```diff
@@ -12,3 +17,42 @@ func (c *Client) CreateGlobalPolicy(name, query, description, resolution, platfo
 	var responseBody globalPolicyResponse
 	return c.authenticatedRequest(req, verb, path, &responseBody)
 }
+
+// ApplyPolicies sends the list of Policies to be applied to the
+// Fleet instance.
+func (c *Client) ApplyPolicies(specs []*fleet.PolicySpec) error {
+	req := applyPolicySpecsRequest{Specs: specs}
+	verb, path := "POST", "/api/latest/fleet/spec/policies"
+	var responseBody applyPolicySpecsResponse
+	return c.authenticatedRequest(req, verb, path, &responseBody)
+}
+
+// GetPolicies retrieves the list of Policies. Inherited policies are excluded.
+func (c *Client) GetPolicies(teamID *uint) ([]*fleet.Policy, error) {
+	verb, path := "GET", ""
+	if teamID != nil {
+		path = fmt.Sprintf("/api/latest/fleet/teams/%d/policies", *teamID)
+	} else {
+		path = "/api/latest/fleet/policies"
+	}
+	var responseBody listGlobalPoliciesResponse
+	err := c.authenticatedRequest(nil, verb, path, &responseBody)
+	if err != nil {
+		return nil, err
+	}
+	return responseBody.Policies, nil
+}
+
+// DeletePolicies deletes several policies.
+func (c *Client) DeletePolicies(teamID *uint, IDs []uint) error {
+	verb, path := "POST", ""
+	req := deleteTeamPoliciesRequest{IDs: IDs}
+	if teamID != nil {
+		path = fmt.Sprintf("/api/latest/fleet/teams/%d/policies/delete", *teamID)
+		req.TeamID = *teamID
+	} else {
+		path = "/api/latest/fleet/policies/delete"
+	}
+	var responseBody deleteGlobalPoliciesResponse
```

Comment:
Nit for clarity `deleteTeamPoliciesResponse` depending if `teamID != nil`? (it works because they have the same members).

Reply:
Yes, this is intended. Actually, why do we have two separate structs if they are the same?

More info: The responseBody struct doesn't have to exactly match -- it just needs to contain the members that I care about extracting.

Reply:
I'd keep them separate Iif for whatever reason we want to add new fields to one or the other.